### ISSUE TITLE
The command `ignite-cli generate model` overwrites the existing model files

### ISSUE THREAD

**Issue by hamzakat** [infinitered/ignite#2354]
Hi everyone.

Today I executed the handy command `ignite-cli generate model MODEL_NAME` and used an existing model's name, and that's resulted in a direct overwriting, so the codes in the file gone! Luckily, there weren't a lot of code lines so it wasn't a frustrating loss. However, it would be very helpful to implement a check before executing the boilerplate magical generation, and I would be happy to help implementing that if anyone provides a hint!

Thanks in advance 🔥

**Comment by jamonholmgren** [comment]
Oof, yes, that isn't good!

Right now, the output looks like this:

<img width="757" alt="CleanShot 2023-01-13 at 10 47 21@2x" src="https://user-images.githubusercontent.com/1479215/212396075-bff23b1c-2a45-444e-ae0b-d9cd6a587842.png">

I would implement a check (around [here](https://github.com/infinitered/ignite/blob/cd50d094190ead6a6c00739bbb4e505a6d03be64/src/tools/generators.ts#L278-L284)) for `filesystem.exists(...)`, and if it does, output something like this:

```
npx ignite-cli g model Dup


   Generated new files:
   <none>
   
   Skipped these files because they already exist:
   /Users/jh/Code/Temp/GenDouble/app/models/Dup.test.ts
   /Users/jh/Code/Temp/GenDouble/app/models/Dup.ts

  To overwrite these files, run the command again with the `--overwrite` flag
```

This will take a new [GeneratorOptions](https://github.com/infinitered/ignite/blob/cd50d094190ead6a6c00739bbb4e505a6d03be64/src/tools/generators.ts#L216-L219) flag, `overwrite: boolean`.

You will also probably need to change the default return value (`Promise<string>`) to something like `Promise<{ written: string[], overwritten: string[], exists: string[] }>` so you can output the right info at the end.

Thoughts?

**Comment by frankcalise** [comment]
Closed by #2363
